float/src/main/java/com/novoda/floatschedule/convert/FloatDateConverter.java

```diff
@@ -2,19 +2,31 @@
 
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
 import java.util.Date;
+import java.util.TimeZone;
 
 public class FloatDateConverter {
 
-    private static final SimpleDateFormat FLOAT_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
+    private static final String FLOAT_DATE_FORMAT = "yyyy-MM-dd";
+    private static final DateTimeFormatter FLOAT_DATE_TIME_FORMATTER = DateTimeFormatter.ofPattern(FLOAT_DATE_FORMAT);
+    private static final SimpleDateFormat FLOAT_SIMPLE_DATE_FORMATTER = new SimpleDateFormat(FLOAT_DATE_FORMAT);
     private static final Date NO_DATE = null;
     private static final String NO_STRING = null;
 
-    public String toFloatDateFormat(Date date) {
+    public String toFloatDateFormat(Date date, String timezoneName) {
```

Comment:
None, I am using timezoneName since in this method only we use the `TimeZone` class. I could expose `TimeZone` right from the beginning and pass that one across methods, if you think that's best.